apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts

```diff
@@ -0,0 +1,28 @@
+import { Address, PublicClient } from 'viem'
+import { useQuery } from '@tanstack/react-query'
+import { getBoostedPoolApr } from '@pancakeswap/pools'
+
+interface UseBoostedPoolApr {
+  client: PublicClient
+  sousId: number
+  contractAddress: Address
+  chainId: number | undefined
+}
+
+export const useBoostedPoolApr = ({ client, sousId, contractAddress, chainId }: UseBoostedPoolApr): number => {
```

Comment:
if this hook rest within `apps/web` then there's no need to add client as parameter. We can import from utils directly.

But if this hook is placed under `pacakges/pools` then we need to feed in the client as one of the params